public/index.html

```diff
@@ -16,6 +16,7 @@
         </div>
       </div>
     </main>
+    <script src="/bower_components/fxa-relier-client/fxa-relier-client.min.js"></script>
```

Comment:
I'm not sure about that. It's more code to load on the buy flow and direct linking is not expected afaik.
Can we not just hard fail early if conditions aren't met?